playwright/e2e/paging.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { Page } from '@playwright/test';
+import { expect, test } from '../support/test-helpers';
+
+test.describe('paging', () => {
+  test.describe('client side', () => {
+    const example = 'client-paging';
+
+    test(example, async ({ si, page }) => {
+      await si.visitExample(example);
+      await pagerTest(page, 10);
+
+      await si.runVisualAndA11yTests('default-paginator');
+    });
+  });
+
+  test.describe('server side', () => {
+    const example = 'server-paging';
+
+    test(example, async ({ si, page }) => {
+      await si.visitExample(example);
+
+      await page.waitForSelector('datatable-row-wrapper');
+      await pagerTest(page, 20);
+      await expect(page.locator('.empty-row')).not.toBeVisible();
+      await page.waitForSelector('span[title="Tonya Bray"]');
+      await expect(page.getByRole('cell', { name: 'Ethel Price' })).not.toBeVisible();
+      await si.runVisualAndA11yTests('server-side-paginator');
+    });
+  });
+
+  test.describe('paging scrolling with no virtualization', () => {
+    const example = 'paging-scrolling-novirtualization';
+
+    test(example, async ({ si, page }) => {
+      await si.visitExample(example, false);
```

Comment:
```suggestion
      await si.visitExample(example);
```

the autoscaling was removed, so please adjust every call that was setting this to false.